src/components/CreateTags/index.js

```diff
@@ -0,0 +1,58 @@
+import {useState} from 'react';
+
+import useStore from '../../hooks/useStore';
+import {Form, FormRow} from '../Form/styled';
+
+export default function CreateTags({selectedPodcast}) {
+	const addTagToPodcast = useStore(state => state.addTagToPodcast);
+	const addTag = useStore(state => state.addTag);
+	const tags = useStore(state => state.tags);
+	let [duplicateTags, setDuplicateTags] = useState([]);
+
+	function handleSubmit(event) {
+		event.preventDefault();
+
+		// reset duplicateTags array every time the form is submitted
+		duplicateTags = [];
+		setDuplicateTags(duplicateTags);
+
+		const input = event.target.elements.tags.value;
+
+		// avoid creation of empty tags
+		if (input.length === 0) {
+			return;
+		}
+
+		const newTags = input.split(',').map(tag => tag.trim());
+
+		newTags.forEach(tag => {
+			// check if the tag already exists
+			if (!selectedPodcast.tags.includes(tag) && !tags.includes(tag)) {
+				addTagToPodcast(tag, selectedPodcast);
+				addTag(tag);
+			} else {
+				duplicateTags.push(tag);
+			}
+		});
```

Comment:
Maybe you might add a if-block to avoid empty tags, something like this:

```suggestion
		newTags.forEach(tag => {
		if(tag.length){
			// check if the tag already exists
			if (!selectedPodcast.tags.includes(tag) && !tags.includes(tag)) {
				addTagToPodcast(tag, selectedPodcast);
				addTag(tag);
			} else {
				duplicateTags.push(tag);
			}
			}
		});
```

Reply:
I was like: Oh, but I already check for empty tags. Than it hit me - I don't! I only checked for an empty input. Your solution is much better.